registrar/apps/core/utils.py

```diff
@@ -153,14 +153,16 @@ def load_records_from_csv(csv_string, field_names):
     """
     row_iter = csv_string.splitlines()
     reader = StrippedLowercaseFieldNamesDictReader(row_iter)
-    missing_fields = field_names - set(reader.fieldnames)
+    required_fields = field_names - set(optional_fields)
```

Comment:
(ALTERNATIVE SOLUTION)

If using `frozenset` doesn't work...

Do `def load_records_from_csv(csv_string, field_names, optional_fields=None):` in the function signatures,
and then here, do:
```suggestion
    required_fields = field_names - (optional_fields or set())
```